docs/source/extra_configuration.rst

```diff
@@ -1,8 +1,19 @@
+.. _extra-configuration:
+
 Extra Configuration
 ===================
 
-To create a new experiment-specific configuration variable, define
-``extra_parameters`` in your ``experiment.py`` file:
+To create a new experiment-specific configuration variable, you can override
+the :attr:`~dallinger.experiment.Experiment.extra_parameters` method on your
+custom Experiment class:
+
+::
+
+    def extra_parameters(self):
+        config = get_config()
+        config.register('n', int, [], False)
+
+Alternatively you can define an ``extra_parameters`` function in your ``experiment.py`` file:
```

Comment:
"Alternatively" suggests that it's one or the other, but it seems like you can use both routes, right? Maybe useful to clarify that this is the case.

Reply:
Indeed, you can use both or either. So maybe "Additionally"?

Reply:
Yes, you could perhaps write:

> Additionally you can define an ``extra_parameters`` function in your
``experiment.py`` file, and both will be respected: